src/chariot/database/DatabaseWriter.py

```diff
@@ -0,0 +1,61 @@
+import abc
+from typing import List
+
+
+def checkDataPoint(dataPoint: dict):
```

Comment:
I see no reason why this is not an instance method of the `DatabaseWriter` class.